app/javascript/tests/public_health/actions/UpdateCaseStatus.test.js

```diff
@@ -0,0 +1,98 @@
+    const mountedWrapper = getMountedWrapper();
+    const option = ''
+    mountedWrapper.find('FormControl').first().simulate('change', { target: { id: 'case_status', type: 'change', value: option } })
+    expect(mountedWrapper.find('.modal-body').childAt(0).exists()).toBeTruthy()
+    expect(mountedWrapper.find('.modal-body').childAt(1).exists()).toBeTruthy()
+    // When options other than the empty string ('') are chosen, this element will have more than 2 children
+    expect(mountedWrapper.find('.modal-body').childAt(2).exists()).toBeFalsy()
+  });
+
+  it('Properly sets the correct follow-up options for "Confirmed", and "Probable"', () => {
+    // have to `mount` the wrapper so the event object is properly created when simulating change below
+    const mountedWrapper = getMountedWrapper();
+    // "Confirmed" and "Probable" have identical behavior in the modal
+    ['Confirmed', 'Probable'].forEach(case_status_option => {
+      mountedWrapper.find('FormControl').first().simulate('change', { target: { id: 'case_status', type: 'change', value: case_status_option } })
+
+      // We expect to find a second FormControl now, containing more options
+      expect(mountedWrapper.find('p').at(1).text()).toContain('Please select what you would like to do:')
+      expect(mountedWrapper.find('FormControl').at(1).exists()).toBeTruthy()
+      const CASE_STATUS_FOLLOW_UP_OPTIONS = ['', 'End Monitoring', 'Continue Monitoring in Isolation Workflow']
+      mountedWrapper.find('FormControl').at(1).find('option').forEach((option, index) => {
+        expect(option.text()).toEqual(CASE_STATUS_FOLLOW_UP_OPTIONS[index]);
+      });
+      expect(mountedWrapper.find('FormGroup').at(0).exists()).toBeTruthy()
+
+      let follow_up_options = [
+        {formControlOption: 'End Monitoring', pText: 'The selected monitorees will be moved into the "Closed" line list, and will no longer be monitored.' },
+        {formControlOption: 'Continue Monitoring in Isolation Workflow', pText: 'The selected monitorees will be moved to the isolation workflow and placed in the requiring review, non-reporting, or reporting line list as appropriate.' }
+      ]
+      follow_up_options.forEach(fuo => {
+        mountedWrapper.find('FormControl').at(1).simulate('change', { target: { id: 'follow_up', type: 'change', value: fuo.formControlOption } })
+        // console.log(`-------------------------------------- ${fuo.formControlOption} --------------------------------------`)
+        // console.log(mountedWrapper.state())
+        // console.log(mountedWrapper.debug())
+        expect(mountedWrapper.state('follow_up')).toEqual(fuo.formControlOption)
+        expect(mountedWrapper.find('p').at(2).text()).toContain(fuo.pText)
+        // expect(mountedWrapper.find('FormCheckInput').simulate('change', { target: { id: 'follow_up', type: 'checkbox', value: fuo.formControlOption } })
```

Comment:
Do these lines of commented out code need to be saved?\

Reply:
I wanted to get this up for UAT, they'll be squashed before this is ready for a full review!

Reply:
Should I wait to continue reviewing this PR or are there just certain files I should avoid looking at for now?

Reply:
Feel free to review the non-test files. The behavior might change following UAT, but the front-end should be good for review now!